src/senaite/core/exportimport/genericsetup/structure.py

```diff
@@ -137,9 +137,13 @@ def _initUsers(self, context, node):
                 user_id = cn.firstChild.nodeValue
                 user = api.user.get_user(user_id)
                 if not user:
-                    self._logger.info("Adding user {}".format(user_id))
-                    # add a new user with the same password as the user id
-                    user = reg_tool.addMember(user_id, user_id)
+                    # add a new user with the same password as the user id or "password" if <5 chars
+                    if len(user_id) >= 5:
+                        password = user_id
+                    else:
+                        password = 'password'
+                    self._logger.info("Adding user {} with password {}".format(user_id,password))
+                    user = reg_tool.addMember(user_id, password) 
```

Comment:
Hi Inus,
would you mind to make the default password just `12345` for just all cases instead?
This makes it easier to remember for all users after import.
Thanks

Reply:
Hi Ramon, I made it so. Also, added a fix so export -> import (in new site) works

Reply:
make it just for all users like this:

```python
 if not user:
    user = reg_tool.addMember(user_id, '12345') 
 ```